### PR TITLE
Improvements to quantizer: Removed unused qType field, add reshape op

### PR DESCRIPTION
**Pull request by pranav-prakash**
* Remove the unused `qType` field. This was always set to `uint8` anyway, which was incorrect for `int8` quantization.
* Fix case when quantized bias already present in quantizer map
* Support for Reshape op added, which is a passthrough (no-op in terms of quantization).

### CODE REVIEW

File: onnxruntime/python/tools/quantization/operators/reshape.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import onnx
```
Comment (yufenglee) [inline onnxruntime/python/tools/quantization/operators/reshape.py:1]: nice! could you add an unit test for it?

Comment (pranav-prakash) [reply onnxruntime/python/tools/quantization/operators/reshape.py:1]: Added!

Comment (yufenglee) [reply onnxruntime/python/tools/quantization/operators/reshape.py:1]: Thanks for your contribution! LGTM overall. Could you please remove the debugger break point in the unit test line 79?

Comment (pranav-prakash) [reply onnxruntime/python/tools/quantization/operators/reshape.py:1]: @yufenglee Fixed, sorry about that.

**Review by yufenglee — DISMISSED** [review]

**Comment by yufenglee** [comment]
@pranav-prakash , it fails some unit tests. Could you please fix them?

**Comment by pranav-prakash** [comment]
@yufenglee Fixed. As part of fixing that I saw that in `quant_utils` the type of `qType` was inconsistent: in all other places `qType` is of type `onnx_proto.TensorProto`, but in `quantize_nparray` it was expected to be of type `QuantType`. This meant that there was actually a latent bug in the existing `pad.py` because in line 42 we had called

```
quantize_nparray(quantized_input_value.qType, padding_constant_array, scale_value, zp_value)
```

but then in line 52 we call

```
pad_value_qnodes = self.quantizer._get_quantize_input_nodes(node, 2, quantized_input_value.qType, ...
```

and `_get_quantize_input_nodes` expects `qType` to be of type `onnx_proto.TensorProto`.

I've cleaned it up so everything is consistent in using ``onnx_proto.TensorProto`.

**Review by yufenglee — APPROVED** [review]

**Pull request merged**